src/Cli/InvokeCommand.php

```diff
@@ -42,12 +43,33 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $simpleHandler = ($this->invokerLocator)();
 
         $event = [];
-        if ($input->getOption('event')) {
-            $event = json_decode($input->getOption('event'), true);
+        if ($option = $input->getOption('event')) {
+            $event = json_decode($option, true);
             if ($event === null) {
-                throw new \RuntimeException('The `--event` option provided contains invalid JSON: ' . $input->getOption('event'));
+                throw new \RuntimeException('The `--event` option provided contains invalid JSON: ' . $option);
             }
         }
+        if ($option = $input->getOption('path')) {
+            $path = $option;
+            if (0 !== strpos($option, '/') && false === (bool)preg_match('#^[a-z]:#i', $path)) {
```

Comment:
Does that mean that I cannot use an absolute path? If so explain that in a code comment? (it's not obvious on first read)

And is there a reason we want to limit that?

Reply:
That mean opposite, in case of an absolute, we don't need to do resolve that path , so we are going to just continue the treatment. But after all, doing that look useless. Maybe just realpath is enough ?

```suggestion
           if (!$path = realpath($option)) {
                throw new \RuntimeException('The `--path` option is an invalid path: ' . $option);
            }
```

Reply:
Is outdated due to changes.